examples/brightness_pixel_location.py

```diff
@@ -1,32 +0,0 @@
-"""
-============================
```

Comment:
I think I am 👎 on removing this example. I know that it's basically the same as the other one, but it's a super simple reference example on over plotting coordinates.

Reply:
FINE